main/src/cgeo/geocaching/storage/LocalStorage.java

```diff
@@ -181,6 +185,13 @@ public static File getOfflineLogImageDir(final String geocode) {
         return dir;
     }
 
+    @NonNull
+    public static File getMapThemeInternalDir() {
+        final File dir = new File(getGeocacheDataDirectory("shared"), MAP_THEME_INTERNAL_DIR_NAME);
```

Comment:
IIRC there's a app cache directory (not accessible from outside) which could be used as well and AFAIK provides even slightly faster file access. It is made for such caching purposes and can be cleared by the system if storage space runs out (which would IMHO indeed makes sense and is fine). Don't know if it fulfills our needs though... 🤷‍♂️

`context.getCacheDir()` or something like that

Reply:
Hm I just read documentation of context.getCacheDir(). I would not really like to use it since it means to let the system take over control upon what's in it (in the sense that the system may delete content in it any time) and current implementation strongly relies on knowing exactly what is currently inside this dir.

Storage space should not be an issue imho:
* I think it can be expected that a user does not keep e.g. 2-3 different map theme bundles (either zip or extracted) on his device. 
* OpenAndroMap's `Elevate.zip` as an example has roughly 440KB
* -> imho its fair to assume that there should not be more data than e.g. 2-3MB in this dir. 
* For Offline Log images we recently decided to store raw images in the very same place- (see #9806), and even one such raw image takes more space than this (on my device about 3.5-4MB for a standard photo) 
* -> I think storage space argument can be neglected **or** we really have to rethink the "scale image upon upload" decision for Offline Log images

Reply:
As I said, that was just such an idea that came to mind. You'll probably know best what's the way to go for this feature! 

Feel free to merge :-)